apps/shipments/templates/email/shipment_link.mjml

```diff
@@ -0,0 +1,64 @@
+{% load mjml %}
+{% block subject %}shipchain.io - {{username}} shared a shipment details page with you.{% endblock %}
```

Comment:
Is this subject line necessary? When doing so through Django, you actually pass through the subject text. This was added for in Profiles when we were using the Djoser version of email. It probably won't make that much of a difference but couldn't hurt to double check.